imap_processing/ultra/l1c/ultra_l1c_pset_bins.py

```diff
@@ -145,3 +148,29 @@ def get_histogram(
     )
 
     return hist
+
+
+def get_pointing_frame_exposure_times(
+    constant_exposure: Path, n_spins: int, sensor: str
+) -> NDArray:
+    """
+    Compute a 2D array of the exposure.
+
+    Parameters
+    ----------
+    constant_exposure : Path
+        Path to file containing constant exposure data.
+    n_spins : int
+        Number of spins per pointing.
+    sensor : str
+        Sensor (45 or 90).
+
+    Returns
+    -------
+    exposure : np.ndarray
+        A 2D array with dimensions (az, el).
```

Comment:
Is it assumed that the shape of this 2D array will always match the pset bins in azimuth and elevation? I could see it being beneficial to read in the data and interpolate to whatever `(az, el)` grid is passed in but that may never be necessary.

Reply:
It will always match and therefore interpolation will not be required. Good idea though!